client/src/components/Schedule/EventDetails.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import { useLocalStorage } from 'react-use';
+import Link from 'next/link';
+import { Row, Col, Typography, Tooltip, Button } from 'antd';
+import { CloseOutlined } from '@ant-design/icons';
+import moment from 'moment-timezone';
+import css from 'styled-jsx/css';
+import { CourseEvent } from 'services/course';
+import { DEFAULT_COLOR } from '../UserSettings/userSettingsHandlers';
+import { renderTagWithStyle, tagsRenderer } from 'components/Table';
+import { GithubUserLink } from '../GithubUserLink';
+
+const { Title, Text } = Typography;
+
+type Props = {
+  eventData: CourseEvent;
+  alias: string;
+  isPreview?: boolean;
+};
+
+const EventDetails: React.FC<Props> = ({ eventData, alias, isPreview }) => {
+  const [storedTagColors] = useLocalStorage<object>('tagColors', DEFAULT_COLOR);
+  const { event, dateTime, place, organizer, special, duration } = eventData;
+
+  return (
+    <>
+      <div className="container">
+        <Row justify="center" align="middle" gutter={[40, 8]}>
+          <Col>
+            <Title>{event.name}</Title>
+          </Col>
+        </Row>
+
+        {dateTime && (
+          <Row justify="center" align="middle" gutter={[40, 8]}>
+            <Col>
+              <Title level={3}>{moment(dateTime).format('MMM Do YYYY HH:mm')}</Title>
+            </Col>
+          </Row>
+        )}
+
+        {event.type && (
+          <Row justify="center" align="middle" gutter={[24, 20]}>
+            <Col>{renderTagWithStyle(event.type, storedTagColors)}</Col>
+            {special && <Col>{!!special && tagsRenderer(special.split(','))}</Col>}
+          </Row>
+        )}
+
+        {organizer && organizer.githubId && (
```

Comment:
looks like it should be `taskOwnerId`. 
![image](https://user-images.githubusercontent.com/14111020/107760627-24a19780-6d3b-11eb-8e2b-d3ab09624e21.png)